### DISCUSSION TITLE
Unable to Locate 'lakefile.lean'

### DISCUSSION THREAD

**Discussion by anonymous-user803** [lean-dojo/LeanCopilot#131] [Q&A] [answered]
Under 'Using Lean Copilot in Your Project', the first step is:

1. Add the package configuration option moreLinkArgs := #["-L./.lake/packages/LeanCopilot/.lake/build/lib", "-lctranslate2"] to lakefile.lean.

and then
2. Add the following line to lakefile.lean, including the quotation marks:

My concern is I am unable to locate the lakefile.lean that it is trying to tell. There is one lakefile.lean in 'batteries' and another one in 'Mathlilb'. The generic one I can find is lakefile.toml. Could anyone please help?

**Comment by Peiyang-Song** [comment, marked as answer]
Are you trying to use Lean Copilot in mathlib? Most Lean projects have a `lakefile.lean` in their root directory, while mathlib (among some other projects) uses a `lakefile.toml`. No matter it is in a `toml` or a `lean` form, the lakefile you would want to edit is the one that is in the root directory of the project on which you wanted to use Lean Copilot. The latest version of readme elaborates on what you would do in both cases of `lakefile.lean` or `lakefile.toml`. You can follow the readme and pick the option that fits your need.
